### PR TITLE
Set correct labels to EtcdProxy ReplicaSets and Services

### LINKED ISSUE #54: EtcdProxyController Services are pointing to all etcd-proxy pods instead to specific pods
We're creating Services in the EtcdProxyController namespace (in order to expose etcd to API Servers) with the `controller: epc` selector. However, in case we have multiple EtcdStorages/EtcdProxy pods, all services point to all etcd proxy pods.

Expected behavior is that a Service points only to relevant EtcdProxy pods, e.g. if we have `etcd-sample-apiserver` service, it points only to `sample-apiserver` EtcdProxy pods.

### PR DESCRIPTION
**Pull request by xmudrii**
Closes #54

### CODE REVIEW

File: pkg/controller/etcdproxy/helpers.go [hunk 1]
```diff
@@ -19,7 +19,8 @@ func newReplicaSet(etcdstorage *etcdstoragev1alpha1.EtcdStorage,
 	etcdControllerNamespace, etcdProxyNamespace, etcdProxyImage,
 	etcdCoreCAConfigMapName, etcdCoreCertSecretName string, etcdCoreURLs []string) *appsv1.ReplicaSet {
 	labels := map[string]string{
-		"controller": "epc",
+		"controller": "etcdproxy",
```
Comment (sttts) [inline pkg/controller/etcdproxy/helpers.go]: aren't there standard labels used elsewhere?

Comment (xmudrii) [reply pkg/controller/etcdproxy/helpers.go]: You mean the `epc` label? No, it's used nowhere, but I was putting it to be used by service.
As it's not used by service at all, should we remove it maybe?

Comment (deads2k) [reply pkg/controller/etcdproxy/helpers.go]: > aren't there standard labels used elsewhere?

@sttts I don't know what you're referring to either.

File: pkg/controller/etcdproxy/helpers.go [hunk 2]
```diff
@@ -134,7 +135,8 @@ func newReplicaSet(etcdstorage *etcdstoragev1alpha1.EtcdStorage,
 
 func newService(etcdstorage *etcdstoragev1alpha1.EtcdStorage, etcdControllerNamespace string) *corev1.Service {
 	labels := map[string]string{
-		"controller": "epc",
+		"controller": "etcdproxy",
+		"apiserver":  etcdstorage.Name,
```
Comment (deads2k) [inline pkg/controller/etcdproxy/helpers.go]: why do you need the controller label here?  Seems like the apiserver is selective enough.

Comment (xmudrii) [reply pkg/controller/etcdproxy/helpers.go]: Updated. Not needed at all.

File: pkg/controller/etcdproxy/helpers.go (hunk 1 above)
Comment (sttts) [reply pkg/controller/etcdproxy/helpers.go]: Some standardized label or annotation to express which controller this belongs to.

**Pull request merged**